pkg/kn/commands/types.go

```diff
@@ -183,8 +188,8 @@ func (params *KnParams) GetClientConfig() (clientcmd.ClientConfig, error) {
 
 	paths := filepath.SplitList(params.KubeCfgPath)
 	if len(paths) > 1 {
-		return nil, fmt.Errorf("Can not find config file. '%s' looks like a path. "+
+		return nil, fmt.Errorf("can not find config file. '%s' looks like a path. "+
```

Comment:
Error messages are changed due to lint warning "Error string should not be capitalized or end with punctuation".